Resources/Prototypes/_Backmen/Roles/Jobs/Medical/virologist.yml

```
@@ -0,0 +1,42 @@
+- type: job
+  id: BkmVirologist
+  name: job-name-BkmVirologist
+  description: job-description-BkmVirologist
+  playTimeTracker: JobBkmVirologist
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Medical
+      time: 7200 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)